src/pages/organisationUnits/list/OrganisationUnitList.tsx

```diff
@@ -0,0 +1,327 @@
+} from './useOrganisationUnits'
+
+export type OrganisationUnitListItem = Pick<
+    OrganisationUnit,
+    'id' | 'displayName' | 'access' | 'path' | 'level' | 'parent'
+> & {
+    ancestors: OrganisationUnitListItem[]
+    hasChildren?: boolean
+    childCount: number
+}
+
+const useColumns = () => {
+    const { columns: selectedColumns } = useModelListView()
+    const schema = useSchemaFromHandle()
+
+    const columnDefinitions: ColumnDef<OrganisationUnitListItem>[] = useMemo(
+        () =>
+            selectedColumns.map((descriptor) => {
+                return {
+                    accessorKey: descriptor.path,
+                    header: descriptor.label,
+                    cell: ({ row }) => {
+                        return (
+                            <ModelValue
+                                path={
+                                    descriptor.path === 'name'
+                                        ? 'displayName'
+                                        : descriptor.path
+                                }
+                                schema={schema}
+                                sectionModel={row.original}
+                            />
+                        )
+                    },
+                }
+            }),
+        [selectedColumns, schema]
+    )
+
+    return {
+        columnDefinitions,
+        selectedColumns,
+    }
+}
+
+export const OrganisationUnitList = () => {
+    const { columnDefinitions, selectedColumns } = useColumns()
+    const [identifiableFilter] = useSectionListFilter('identifiable')
+
+    const userRootOrgUnits = useCurrentUserRootOrgUnits()
+    const userRootOrgUnitIds = useMemo(
+        () => userRootOrgUnits.map((ou) => ou.id),
+        [userRootOrgUnits]
+    )
+
+    const schema = useSchema(SchemaName.organisationUnit)
+
+    // the expanded organisationUnit Ids
+    // note that this controls which orgUnits we load data for through usePaginatedChildrenOrgUnitsController
+    const [expanded, setExpanded] = useState<ExpandedState>(() =>
+        Object.fromEntries(userRootOrgUnitIds.map((ouId) => [ouId, true]))
+    )
+
+    // we keep a diferent state for the expanded org units during filtering
+    // because we want to expand all ancestors - but do NOT want to load data through usePaginatedChildrenOrgUnitsController
+    // since we already get the data we need through useFilteredOrgUnits
+
+    const [expandedDuringFilter, setExpandedDuringFilter] =
+        useState<ExpandedState>({})
+
+    const fieldFilters = selectedColumns.map((column) =>
+        getFieldFilter(schema, column.path)
+    )
+
+    const orgUnitFiltered = useFilteredOrgUnits({
+        searchQuery: identifiableFilter,
+        fieldFilters,
+        enabled: !!identifiableFilter,
+    })
+    const isFiltering = !!identifiableFilter && !orgUnitFiltered.isIdle
+
+    const { queries, fetchNextPage } = usePaginatedChildrenOrgUnitsController({
+        parentIds: Object.keys(expanded),
```

Comment:
thinking that this would render nothing if expanded is empty. Maybe we could revert back to the default in that case? Or not sure if you have other ideas?

e.g. `parentIds: Object.keys(expanded).length === 0 ? userRootOrgUnitIds : Object.keys(expanded),`

Reply:
Actually on second thought, i think the top level should always be there, no? So more something like: 

` parentIds: userRootOrgUnitIds.concat(Object.keys(expanded)).someDeduplicateFun()`

Reply:
Good catch! I think ```   parentIds: Object.keys({ ...expanded, ...inititalExpandedState })``` should suffice. I may want to memo this tho, so the memoed `flatParentIdPages` isn't recomputed everytime?

Reply:
Yes, much better than what i wrote! Thanks :)

Reply:
It's actually a bit more complicated because that would always show the children of the root (even when filtering)...

Reply:
Eg. 
```
 const parentIdsToLoad = useMemo(() => {
        if (isFiltering) {
            return Object.keys(expanded)
        }
        // when we are not filtering we always want to load the root org units, so that the table is never empty
        return Object.keys({ ...inititalExpandedState, ...expanded })
    }, [isFiltering, expanded, inititalExpandedState])
```

I dont love the extra complexity and different cases when "isFiltering", but Im not seeing a better way right now

Reply:
ah i see you are right. I think this reads fine